data/paths.json

```diff
@@ -3,5 +3,6 @@
   "/_next/image": "nextjs",
   "/_next/static": "nextjs",
   "/_vercel/image": "vercel",
-  "/is/image": "scene7"
+  "/is/image": "scene7",
+  "/assets": "directus"
```

Comment:
I'm concerned that this is a bit too generic to use as a path. There could be a lot of false positives. We might need to stick with manually specifying the CDN

Reply:
Done! I have removed that code to avoid false positives

Reply:
Thanks